docs/source/modules/mixedlayer.rst

```diff
@@ -0,0 +1,277 @@
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+|``update_albedo_from_ice``     | Flag to set the surface albedo to ``ice_albedo_value`` where there is ice as specified by ``ice_file_name`` | ``False``            |
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+|``ice_albedo_value``           | Value for ice albedo when ``update_albedo_from_ice`` is True.                                               | ``0.7``              |
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+|``ice_file_name``              | Name of file containing sea ice concentration.                                                              | ``siconc_clim_amip`` |
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+|``ice_concentration_threshold``| Value of sea ice concentration above which albedo should be set to ``ice_albedo_value``.                    | ``0.5``              |
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+|``ice_file_name``              | Name of file containing sea ice concentration.                                                              | ``siconc_clim_amip`` |
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+|``do_ape_sst``                 | Flag to set prescribed SST according to the APE aquaplanet analytic form                                    | ``False``            |
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+|``add_latent_heat_flux_anom``  | Flag to add an anomalous latent heat flux                                                                   | ``False``            |
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+
+Diagnostics
+-------------------
++-------------------+-------------------------------------+------------------------------------+
+| Name              | Description                         | Units                              |
++===================+=====================================+====================================+
+| t_surf            | Surface temperature                 | :math:`\text{K}\`                  |
```

Comment:
t_surf and delta_t_surf have not rendered properly. I can't see any bug here but perhaps it is the last '\'.